testsuites/lbt-prelude/lbt-prelude-typescript/.envrc

```diff
@@ -0,0 +1,2 @@
+nix_direnv_watch_file build.json
```

Comment:
This command is not available in new direnv for nix. What's build.json btw?

Reply:
Oh, I didn't know
I'll remove these